Sources/Beacon.Core/Config.cs

```diff
@@ -28,5 +28,15 @@ public class Config
         /// Check the build status using the TeamCity guest account.
         /// </summary>
         public bool GuestAccess { get; set; }
+
+        /// <summary>
+        /// Monitor builds for active branches only.
+        /// </summary>
+        public bool ActiveBranchesOnly { get; set; }
```

Comment:
🤔 Shouldn't this be the default?

Reply:
Since monitoring only active branches closer matches TeamCity's behavior, I think it should. But then the option needs to change. Maybe into something like `IncludeAllBranches` or `IncludeInactiveBranches`?